framework/include/base/AppFactory.h

```diff
@@ -125,6 +130,8 @@ class AppFactory
 protected:
   AppFactoryBuildInfoMap _name_to_build_info;
 
+  std::map<std::string, size_t> _app_creation_count;
```

Comment:
This should just live in the AppFactoryBuildInfoBase

Reply:
Done